internal/exec/resolvable/resolvable.go

```diff
@@ -380,13 +384,46 @@ func findMethod(t reflect.Type, name string) int {
 	return -1
 }
 
-func findField(t reflect.Type, name string) int {
+func findField(t reflect.Type, name string, index []int) []int {
```

Comment:
Why does this method need an integer array? `, index []int`?

Reply:
This function builds the index sequence for [`Type.FieldByIndex`](https://golang.org/pkg/reflect/#Value.FieldByIndex) recursively, returning the sequence if found and an empty array otherwise.